docs/corpora_schema.md

```diff
@@ -34,100 +35,110 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
+
+Ontology terms MUST use [OBO-format ID](http://www.obofoundry.org/id-policy.html), meaning they are a CURIE
+where the prefix identifies the ontology. For example `EFO:0000001` is a term in the `EFO` ontology.
 
 **Field name**|**Constraints**
 :--|:--
-tissue\_ontology\_term\_id|UBERON term
+tissue\_ontology\_term\_id|UBERON term. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
 assay\_ontology\_term\_id|EFO term
 disease\_ontology\_term\_id|MONDO term or [PATO:0000461](http://bioportal.bioontology.org/ontologies/PATO?p=classes&conceptid=PATO%3A0000461)
 cell\_type\_ontology\_term\_id|CL term
 ethnicity\_ontology\_term\_id|HANCESTRO term, "na" if non-human
 development\_stage\_ontology\_term\_id|HsapDv term if human, child of EFO:0000399 otherwise
 
-The `tissue_ontology_term_id` field must be appended with " (cell culture)" or " (organoid)" if appropriate.
+cellxgene requires ontology terms to enable search, comparison, and integration of data. When no appropriate ontology value is available, then the most
+precise accurate term MUST be used. For example if the `cell_type` field describes a relay interneuron, but the most specific available term in the CL
+ontology is CL:0000099 ("Interneuron"), then the interneuron term can be used to fulfill this requirement, and ensures that users searching for "neuron"
+are able to find these data. Users will still be able to access more specific cell type annotations that have been submitted with the
+data (but aren't required by the schema). A dataset comprising cells of the human embryo provides a more extreme example. In this case, the most  
+precise accurate term may be the root of the cell ontology `cell`, or its child term `cell in vitro`. The Cell Ontology is expanding over time, and we hope to
+migrate datasets to more defined terms as they are defined. In the mean time, having these less precise terms maximizes the findability (and therefore reusability)  
+of datasets.
 
-The `ontology_term_id` fields may be empty strings when no appropriate ontology value is available, for example if the
-`cell_type` field describes a cell type for which no ontology term exists or if the `ethnicity` is unknown. If the
-field is not empty, then it must be an [OBO-format ID](http://www.obofoundry.org/id-policy.html), meaning it is a CURIE
-where the prefix identifies the ontology. For example `EFO:0000001` is a term in the `EFO` ontology.
+#### Gene Metadata
 
-If the features of the dataset are human genes, then the feature ids must be [HGNC](https://www.genenames.org/about/guidelines/#!/#tocAnchor-1-7) approved
-gene symbols.
+Cellxgene uses standard gene symbols to ensure that all datasets it stores measure the same features and can therefore be integrated.
+If the samples being measured are human, then the feature ids MUST be [HGNC](https://www.genenames.org/about/guidelines/#!/#tocAnchor-1-7) approved
```

Comment:
A couple questions in regards to gene symbols:
- Is there a specific version of HGNC that you are validating against? For example, can we have EMR2 (a previous symbol) as a gene index rather than the current ADGRE2?
- Is there additional information on how to handle gene symbols that are redundant? Can there be a number appended as a suffix?

Reply:
RE HGNC version - see https://github.com/chanzuckerberg/single-cell-curation/issues/22

Reply:
Also see https://github.com/chanzuckerberg/single-cell-curation/blob/f54de6f710fbe7d66de76bec0249931b87e4930f/cellxgene_schema_cli/cellxgene_schema/gene_symbol.py#L183 - 

 

>  Also, seurat and scanpy append ".1" or "-1" to duplicated gene names, and these altered
>     names persist throughout the life of the object. They won't match against the HGNC database
>     and we want to merge them, so we need to strip off the suffix and try matching again. 

> This function takes a symbol and returns the symbol with the fixed case and also with the
>     seurat/scanpy suffix stripped off.